lib/foodcritic/rules/fc078.rb

```diff
@@ -0,0 +1,135 @@
+rule "FC078", "Ensure cookbook shared under an OSI-approved open source license" do
+  tags %w{supermarket}
```

Comment:
That also makes sense, but has me wondering what the meaning of the `supermarket` tag is. I suppose a `community` tag could be applied to rules that map to [cookbook quality metrics](https://github.com/chef-cookbooks/cookbook-quality-metrics/blob/master/SMQM.md).